tools/runners/VanillaYosysPlugin.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+import os
+import sys
+
+from BaseRunner import BaseRunner
+
+
+class VanillaYosysPlugin(BaseRunner):
```

Comment:
We need to make sure it is clear that it's the UHDM plugin (as there are many different yosys plugins in general).